tcf_website/tests/test_user.py

```diff
@@ -0,0 +1,29 @@
+# pylint: disable=no-member
+# pylint: disable=missing-module-docstring
```

Comment:
Can we have docstring instead? I believe the number of exceptions should be minimal.